nemoguardrails/library/factchecking/align_score/request.py

```
@@ -28,24 +27,30 @@ async def alignscore_request(
+        raise_for_status=False,
+    )
+    if http_response.status_code != 200:
+        log.error(f"AlignScore API request failed with status {http_response.status_code}")
+        return None
+
+    result = http_response.json()
+
+    log.info(f"AlignScore was {result}.")
+    try:
+        result = result["alignscore"]
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)